### PR TITLE
Bau - Stop building on jenkins

### PR DESCRIPTION
**Pull request by ghost**
- We are already building on concourse so stop building with jenkins

### CODE REVIEW

**Review by JohnWattsGDS — COMMENTED** [review]
LGTM

File: build.jenkins [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-
-    stage('Tests') {
-      agent {
-        dockerfile {
-          reuseNode true
-        }
-      }
-      steps {
-        sh 'echo "Running unit tests and integration tests"'
-        sh './gradlew clean test intTest'
```
Comment (bjgill) [inline build.jenkins:24]: > intTest

What is doing this? I can't immediately see anything in the Concourse pipeline that does this step...

File: build.jenkins [hunk 2]
```diff
@@ -1,35 +0,0 @@
...
-      }
-    }
-
-    stage('Run test tool scenarios') {
-      agent any
-      steps {
-        sh './jenkins.sh'
```
Comment (bjgill) [inline build.jenkins:31]: Can `jenkins.sh` be deleted?

**Review by JohnWattsGDS — DISMISSED** [review]

**Review by bjgill — COMMENTED** [review]
I assume that https://deployer.tools.signin.service.gov.uk/teams/main/pipelines/verify-service-provider is the pipeline that covers this?

File: build.jenkins (hunk 2 above)
Comment (ghost) [reply build.jenkins:31]: Yes it can.

File: build.jenkins (hunk 1 above)
Comment (ghost) [reply build.jenkins:24]: Let me check the config.

Comment (ghost) [reply build.jenkins:24]: Good spot. @jakubmiarka Do you know if this need to be running the intTests in concourse - https://github.com/alphagov/verify-terraform/blob/master/environments/tools/platform/files/pipelines/verify-service-provider.yml#L814

Comment (jakubmiarka) [reply build.jenkins:24]: possibly? ... The VSP pipeline is for VSP and we pull the LMS just to use it (to support the e2e journey)... So we should run the intTests somewhere...

Comment (JHjava) [reply build.jenkins]: Got you. We will migrate this to build-libraries on concourse then to build and run the tests.

Comment (ghost) [reply build.jenkins:24]: Ok. We're running these now in Travis.

**Review by JohnWattsGDS — APPROVED** [review]

**Review by dbes-gds — APPROVED** [review]

**Pull request merged**